Reports view

naugle92 opened the pull request:
created a new view: reports

naugle92 committed:
Added Reports view

naugle92 committed:
Merge branch 'master' of https://github.com/3See/etect-app into reports_view

review-ninja commented:
<a href="https://app.review.ninja/3See/etect-app/pull/33" target="_blank"><img src="https://app.review.ninja/43009198/pull/33/badge" alt="ReviewNinja"/>

Blistin commented:
@naugle92 Once Kevin finishes up with #32 please pull down the changes and add the link to get to this page onto the Records button and rename the label on the sidebar to 'Reports'

Blistin commented on modules/core/client/config/core.client.routes.js:
```diff
@@ -48,6 +48,10 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
                 url: '/subject-setup',
                 templateUrl: 'modules/core/views/subject_setup.client.view.html'
+            })
+            .state('reports', {
```
!fix Please remember to pull #32 and get setup this route through the sidebar, rename the 'Records' button to 'Reports'

csturco0331 replied:
!fixed

csturco0331 committed:
Merge branch 'master' of https://github.com/3See/etect-app into reports_view

csturco0331 committed:
Added reports links, removed breadcrumbs, fixed grunt error

added the link to the reports page to the reports button on the sidenav.
Removed outdated breadcrumbs from several page views.
Added semicolon to reportController to fix a grunt error.
